Gopkg.lock

```diff
@@ -3,65 +3,52 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:4c5803754aedbf42ec7da8cdfea47d36337397267851828610a4f9a76c1b7fd9"
```

Comment:
We've actually stopped supporting `dep`, so let's undo this change.  In not too long, we'll remove the Gopkg files and update the instructions to explain using vgo with go1.10

Reply:
ok